cmake/modules/FindFriBidi.cmake

```diff
@@ -17,12 +17,28 @@ if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_FRIBIDI fribidi QUIET)
 endif()
 
+if(CMAKE_HOST_WIN32)
+  find_package(fribidi NO_MODULE)
```

Comment:
Upstream FriBidi doesn't use the CMake buildsystem, so this will never succeed.

Reply:
We don't ever want to find upstream fribidi on windows so this is correct

Reply:
We don't ever want to find upstream fribidi on windows so this is correct

Reply:
Required is FriBidi, not a FriBidi fork.

Reply:
Since when did we stop allowing patching libs?

You wanted the check moved into cmake and not hard coded and I did that. Now you're saying I can't use cmake to find the definitions for fribidi? Why is pkgconfig allowed to do this then?

Reply:
You are allowed to patch libs, but not to require additional configuration files, which aren't generated by the upstream build system.
Just patch the library so it works out of the box, so that the project using it haven't to tell the library if itself was built static or shared.
The pkgconfig file is generated by the upstream build system, so it exists for builds using the upstream code.